File: Interpret Problem.py
def interpret(value, commands, args):
  counter = 0
  for command in commands:

    if command == "+":
      value += args[counter]
    elif command == "-":
      value -= args[counter]
    elif command == "*":
      value *= args[counter]
    else:
      return -1

    counter += 1

  return value

File: Interpret Problem.java
public int interpret(int value, String[] commands, int[] args) {
  int counter = 0;
  for(String command: commands)
    if(command.equals("+"))
      value += args[counter++];
    else if(command.equals("-"))
      value -= args[counter++];
    else if(command.equals("*"))
      value *= args[counter++];
    else
      return -1;

  return value;

}
